.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+
+updates:
+  - package-ecosystem: "docker"
+    directory: "/tests/tck-build-logic/src/main/resources/allowed-docker-images"
+    reviewers:
+      - "matneu"
+    schedule:
+      interval: "weekly"
```

Comment:
Any reason we pick weekly? I think this can be a daily pass, shouldn't take long?

Reply:
Sure, can be daily.